README.md

```diff
@@ -297,6 +297,26 @@ of mangled property names.
 Using the name cache is not necessary if you compress all your files in a
 single call to UglifyJS.
 
+When using `minify()`, you may specify `--name-cache` in the following way:
+
+```javascript
+var options = {
+    mangle: {
+        properties: true,
+    },
+    nameCache: JSON.parse(fs.readFileSync("/tmp/cache.json", "utf8"))
+};
+fs.writeFileSync("part1.js", UglifyJS.minify({
+    "file1.js": fs.readFileSync("file1.js", "utf8"),
+    "file2.js": fs.readFileSync("file2.js", "utf8")
+}, options), "utf8");
+fs.writeFileSync("part2.js", UglifyJS.minify({
+    "file3.js": fs.readFileSync("file3.js", "utf8"),
+    "file4.js": fs.readFileSync("file4.js", "utf8")
+}, options), "utf8");
+fs.writeFileSync("/tmp/cache.json", JSON.stringify(options.nameCache), "utf8");
```

Comment:
Could you put "/tmp/cache.json" in a variable `cacheFileName` at the top of the example so the path is not repeated twice?